### ISSUE TITLE
unannoy not found

### ISSUE THREAD

**Issue by peng051410** [skeeto/.emacs.d#7]
I use you config setup, got error: unannoy not found
Have you forgotten to add the etc directory to the scan list?

**Comment by skeeto** [comment]
tl;dr Run "make" in .emacs.d/ before starting Emacs.

Awhile back I decided I wanted to support multiple versions of Emacs at 
the same time, with the same .emacs.d/, while also byte-compiling 
everything. The biggest problem is that byte-compiled files are not 
necessarily backwards or forwards compatible. This means byte-compiled 
.elc files cannot live alongside their source file in this repository, 
since each version of Emacs needs its own byte-compiled files. Instead 
they're all "installed" under .emacs.d/site-lisp/EMACS-VERSION.

This install step is performed via the Makefile, and so my config won't 
work until you "install" the config under site-lisp/ by invoking make.

    $ cd ~/.emacs.d/
    $ make

This concept is also not compatible with package.el (ELPA, MELPA, etc.), 
so I had to get rid of it. Instead I invented my own Emacs package 
manager called gpkg. It fetches "packages" by cloning upstream git 
repositories. The first time you run make, it will clone a bunch of 
repositories, then check out a specific commit under site-lisp/. The 
repositories are shared by all versions of Emacs, and it only has to 
clone them once.

To compile and install the configuration for another version of Emacs, 
set a different EMACS variable for the Makefile:

    $ make EMACS=/usr/local/bin/emacs26

Both this and the previous build will live in parallel under site-lisp/.

To wipe out all "installed" files, use "mostlyclean".

    $ make mostlyclean

And to blow away even the repositories:

    $ make distclean

**Comment by peng051410** [comment]
I see! Can you add some configuration instructions? This way we can understand and use your configuration more clearly.
